etc/docker/travis/alembic_mysql.ini

```diff
@@ -24,7 +24,7 @@ script_location = lib/rucio/db/sqla/migrate_repo/
 # the 'revision' command, regardless of autogenerate
 # revision_environment = false
 
-sqlalchemy.url = mysql://root:secret@mysql/mysql
+sqlalchemy.url = mysql+pymysql://root:secret@mysql/mysql?charset=latin1
```

Comment:
Why latin1? Shouldn't we use utf

Reply:
For some reason the database schema gets initialised with latin1. Without the specific charset pymysql tries to use utf8mb4 and throws a lot of warnings. 
I also remember that I had the same problem with upgrading MySQL in #2478 where all strings with a length of 3000 were causing errors because the key length was too long with utf8 (3000 * 3 bytes > 3072 max key length)

Reply:
there's no workaround for this problem in mysql/mariadb, so latin1 would be acceptable IMHO